### ISSUE TITLE
[Sybil Report] Cluster of 48 addresses 

### ISSUE THREAD

**Issue by AnonL0** [LayerZero-Labs/sybil-report#1612]
# Reported Addresses

Cluster 1:

0xd7bd50a5a5790b817a5b278c44523bb234bb9d85
0x6bc4de4ef4ed4dfe83481c7db6a05306ea163d05
0xa0b4e414c96cdd955586d1facea2a0e90c6413c6
0x7a77d9981a8380420e99bc76dc0c096865a8b8fb
0xfd8ef9393debce066d28d667072f8641c90ebba8
0x367ff4084001f7d8a5409609b99fa9e63eba7a57
0x5a916ce98073081d7ea43798d19edfe97f2691c3
0x011b5df8b1d91a4b65ae653562a2d6e102f7ba79
0xb35881f17e21b5cfb09dbf272afe7832dc5b5dc3
0xb6de6d3a98101d0f95b885d8c8cd6686808dc433
0xdf752b1ce29c9857435444958feffa46d2b58e17
0x27cecbc29694d8255434917c7cd18ccb9e125bdf
0x8c3399126a1806704630269474645c4c9d7f7c01
0x0ae1805a674baa4445450691d9faf7b8b2af813f
0xa93b6ee679a854b5c93db207a6487bfb8bff474a
0xdc521f1c9114e0d7ad307fd58a4d4b23598401ad
0xebdb7d21fb3617cc6c0ebd89ae63b61d84f05b87
0x7ed0c1ba091b36e2300ea78d046592c285922a0b
0x69301f673697d4adbc17ddce98a04683f35f0fd2
0xc404d3251f300bed4d34bd2e98de5352373ce142
0xa59bf1cbc8a319bc279133e5f6a68a336ddbb9e1
0xf8ff55033f233accb0ea6e8a5b5942f720ab40b0
0x0ec81acd4e22d877a0d54fded5ea375a0bfcd9c4
0x34AE4e76a861c5B946C457f6d4f46737c6F6A031

Cluster 2:

0x6b4d7188CAdA13b8f8F2728613dc7a0DBbF68C35
0xeC01CA82f0e8c7C57fA84a820Cbfd72c385B62ac
0xB2252B51C884C91c315Ed1262A2931d305713ee9
0x24b32C99b0607529a719A1947e00710DA69f44e0
0x0059003d7DEd2b0F32E7d26f84E11CA87C3DDbEf
0x2CD45770ea30F0AdC4AA5e43a22bC5E12BC0B11c
0x2ec945AdC742079458E966EfE04fE9657a15b9cf
0x869F47fC27cDc48049695D3A7456a3653a39Cfc5
0x3209f7358781d3f6A876f03dA45EeF21914cB67f
0xC2E2D071d00a0Fe60068Abb62FD490e37f64bfa9
0xe1fF506E6D7B04BD94Db81A6bdE929DF95C13594
0x720d8cee3DBb60597BDe18D1B0566930c033BEfa
0x45F508e4fE5414fF76805804cCcc8f8Ad75A31FC
0xdE5e6968029b00efef3F85CC47ea0cce72c9032d
0xB26E5f5F10E6b55401ED551F06666dBbee20cc55
0x642266C28AdB29Fb9389Ccc176cbE1b77F899182
0x9B27f6e709207835BEFDedD55CC37A995A42e919
0x3B90570CD40269f7685a95Cf2659f0baa733e2b5
0xe30f5a73117749E9F21392509994bf7B18cf85cB
0xF5e6296c4b17b53499fa36813F407dE3197d258b
0x01855C53deFBdD7349E24D2A734c79baC182219D
0xFED8b17f7E44d5528495A0CdA17E91E5E8daE95B
0xa580181C3FaC45d0042c22f95580d0FcCf063c26
0x98F4bf76b36DeA5c1C2116f29ee8f5F9a0FD7bca

# Description

These addresses are not all connected on-chain. Actually they form 24 clusters of 2 addresses. Let me explain how they are linked. 

Cluster 1: 

**The addresses have the exact same activity stats on LayerZero (verified with https://dune.com/springzhang/layerzero-users-ranking-for-potential-airdrop)**

First interaction with LayerZero within the same 1 minutes for all 23 addresses: 2023-10-17 07:03
Same number of TXs: 48
Same bridged amount: ~$32k 
Same source chains/destination chains/ contracts used: 7 / 7 / 8
Same days/weeks/months of activity: 2 / 2 / 2

**This data can be seen in the dune dashboard by writing 2023-10-17 07:03 in the search box. Here are the screenshots (in 2 parts)
# Detailed Methodology & Walkthrough**

![1](https://github.com/LayerZero-Labs/sybil-report/assets/170123522/2281ae8b-e4f5-4135-aa48-571724f39f73)

![1](https://github.com/LayerZero-Labs/sybil-report/assets/170123522/96d54d7e-9e80-4afe-a369-d1549a33a1e4)

These addresses are part of the Cluster 1

**Each address of Cluster 2 is linked to an address from Cluster 1 because of the same OKX deposit address.**

You can see an example here:

https://platform.arkhamintelligence.com/explorer/address/0x090d8A28e526f377dA753d1320299b080f2a8d63

This is an OKX deposit address. 

0xd7bd50a5a5790b817a5b278c44523bb234bb9d85 (Cluster 1)

and

0x6b4d7188CAdA13b8f8F2728613dc7a0DBbF68C35 (Cluster 2)

both deposited to this same OKX deposit address. 

**Here are all the deposit addresses you need to verify:**

https://platform.arkhamintelligence.com/explorer/address/0x090d8A28e526f377dA753d1320299b080f2a8d63
https://platform.arkhamintelligence.com/explorer/address/0x074d1C04023Bd4a8a26D0E2390dfa79dE9EF4F88
https://platform.arkhamintelligence.com/explorer/address/0x80ac051BA5B76DA81b712540732a8b91Eb00dF09
https://platform.arkhamintelligence.com/explorer/address/0x36B5A260FAdcA3Ccb6e81b5dA94C640fD93C58d3
https://platform.arkhamintelligence.com/explorer/address/0x244A04C4Aea54C0FA67bcFF6980ccb481Ac04D47
https://platform.arkhamintelligence.com/explorer/address/0xDde39A85513Cf89BF1742e2774Aa43900e30695e
https://platform.arkhamintelligence.com/explorer/address/0xDFC5085200BE4416DE67F0e83e7e3d549746A490
https://platform.arkhamintelligence.com/explorer/address/0x1E3AE80679F1c6e96A728c40A646B1eA21Dd39dc
https://platform.arkhamintelligence.com/explorer/address/0xcEF1bD9C9062Dc26B8f967D88FAB3d7a206BDd8E
https://platform.arkhamintelligence.com/explorer/address/0xcd33B8F201213594A214c510609F79Ccc0CADb35
https://platform.arkhamintelligence.com/explorer/address/0x1B9D8322AB5e583A953B6ebE36FB56ba05E11948
https://platform.arkhamintelligence.com/explorer/address/0x3A3A440d55f74479A505e91cFAc56818C40c12bD
https://platform.arkhamintelligence.com/explorer/address/0x58CC84413840895eC10b2054444f51F1b9521419
https://platform.arkhamintelligence.com/explorer/address/0x84a0613b5844D18e984d9B2a650Ae1d857574eee
https://platform.arkhamintelligence.com/explorer/address/0xf78a3dA6C7ea4111799081986b76095D1691cdD5
https://platform.arkhamintelligence.com/explorer/address/0x3Ac3B3Dc73Aff3ABA65452117AB325382bB787Af
https://platform.arkhamintelligence.com/explorer/address/0xe991DAB91be8c42f6a7c73CFBd88E8d60671158f
https://platform.arkhamintelligence.com/explorer/address/0x73AF701A141C873B917741F14e486D2C7232cA6b
https://platform.arkhamintelligence.com/explorer/address/0x6f88D0b8ec725628D1631A6D866C924f2E7F0B20
https://platform.arkhamintelligence.com/explorer/address/0x109FdC1211bAdfefA2AB57FE7f6576d21A7463c0
https://platform.arkhamintelligence.com/explorer/address/0x014dDcA05DB45DB7bce45F963608509F2900DBe3
https://platform.arkhamintelligence.com/explorer/address/0xA4C4FcdE433697a36dF93a802752B103edf9bF1a
https://platform.arkhamintelligence.com/explorer/address/0x8Ac11A8CAb3A4eA574A68ec2A2d94eE5FDd761Fc
https://platform.arkhamintelligence.com/explorer/address/0x3B6486eFE572057Ec207b040b23DB5f11A7d5ab0

**And here the detail on how the addresses are linked:**

![1](https://github.com/LayerZero-Labs/sybil-report/assets/170123522/c56f5fa5-ef60-4a82-80ee-2848f10a3dc8)
![1](https://github.com/LayerZero-Labs/sybil-report/assets/170123522/4fd5fb3f-359b-4a39-911c-2c6489c836c3)

**Methodology**

Cluster 1

I found the addresses on Dune dashboard. https://dune.com/springzhang/layerzero-users-ranking-for-potential-airdrop

I filtered with days/weeks/months of activity by ‘2 / 2 / 2’ to try and catch similar patterns. 

That’s how I saw the list of addresses from Cluster 1, with the exact same stats and created the same minute of the same day (sea Reasoning part).

First interaction with LayerZero within the same 1 minutes for all 23 addresses: 2023-10-17 07:03
Same number of TXs: 48
Same bridged amount: ~$32k 
Same source chains/destination chains/ contracts used: 7 / 7 / 8
Same days/weeks/months of activity: 2 / 2 / 2

Also, all addresses from Cluster 1 have been posted on Twitter in replies from the same Tweet (all posted by different bots). Here are some examples:

https://twitter.com/rigonan27/status/1720057113819660545 (1rst addy)
https://twitter.com/Be53528214Crazy/status/1720057113874190644 (2nd addy)
https://twitter.com/fii_bureng/status/1720057113920352316 (3rd addy) 

… and so on for 22 addresses from Cluster 1.

Cluster 2

As explained before, all addresses from Cluster 2 have a similar activity and each address from Cluster 2 is linked to an address from Cluster 1 because of same OKX deposit address.

Farming methodology for Cluster 1 & 2

All addresses have been funded 7 months ago through OKX by depositing ~$30 in ETH on Linea. 

Then,  they used Clique Portal, Horizon Dex and Pancakeswap

Same behavior for all addresses. 

TLDR: 

1.Cluster 1: 23 addresses with exact same activity and created at the same time (likely a bot)

2. Cluster 2: 23 addresses linked to Cluster 1 thanks to CEX deposit address

3. All addresses from cluster 1 posted the same minute under the same tweet by different bots accounts

# Reward Address (If Eligible)

Please provide any reward to this address: **0x5600E7e08198607D1623320Bc68de403c27e6F20**

Thank you, and long life to LayerZero.
